### PR TITLE
Feature/colorful

### PR DESCRIPTION
**Pull request by ayurmedia**
**Colorful, Copy from User Contributed Mashup**

Very useful mashup, from user-contributed mashups, which should be part of the official mashups. 
I removed font-size change, so that this mashup only changes colors but keeps sizes.
Small bug fixes for baseinfo version and removed bus which is not needed. 
In theory this mashup could be reduced to a custom.css, which was not possible with user contributed mashups. 
It works well together with mashup "cleaner tree view". 
The old screenshot here shows that TP was much cleaner before and now has a lot of columns in the tree/list view which makes it noisy.

### CODE REVIEW

File: Colorful/Colorful.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+tau.mashups
+    .addDependency('tp3/mashups/topmenu')
+    .addMashup(function(topMenu) {
+        'use strict';
+        var initColorfulTheme = function() {
+            var style = document.createElement("style");
+            style.setAttribute('id', 'qs-color-style');
```
Comment (tsarevichdm) [inline Colorful/Colorful.js]: Suggest to change id here in case somebody want to install original colorful mashup / use it as base for its own one.

**Comment by tsarevichdm** [comment]
Cool, thank you!

**Comment by iago849** [comment]
Какого черта я на это подписан

**Comment by iago849** [comment]
Я не работаю на ТР уже 3 года

**Comment by tsarevichdm** [comment]
@iago849 Странно, возможно осталась подписка на репозиторий. Явно тебя не добавлял.

**Comment by tsarevichdm** [comment]
image

**Pull request merged**